tests/dsl_unittest.py

```diff
@@ -98,7 +98,7 @@ def setUp(self):
         reset()
 
     def run_example(self, exapmle):
-        _ExampleRunner(exapmle, QuietFormatter()).run()
+        _ExampleRunner(exapmle, QuietFormatter(import_module_names=__name__)).run()
```

Comment:
Shouldn't this be `[__name__]` because `import_module_names` is expected to be a list?

Reply:
Indeed... mypy should have caught this if we had `def run_example(self, exapmle) -> None:`...